ramls/credentialsExistence.json

```diff
@@ -6,6 +6,11 @@
     "credentialsExist": {
       "type": "boolean",
       "description": "True if user has local password"
+    },
+    "isPlaceholder": {
+      "type": "boolean",
+      "description": "True if credentials record is a placeholder",
+      "default": false
```

Comment:
Documentation what a placeholder credential is, how it can be created and whether it allows to do a successful login is missing from the POST /authn/credentials RAML documentation.